src/plugins/requestAuthPlugin.js

```diff
@@ -24,7 +30,7 @@ function verifyAuth(auth) {
 
 const handleLogout = request => {
 	request.log(['info', 'auth'], 'Logout received, clearing cookies to re-authenticate');
-	return removeAuthState(['oauth_token', 'refresh_token'], request, request.plugins.requestAuth.reply);
+	return removeAuthState(['MEETUP_MEMBER', 'oauth_token', 'refresh_token'], request, request.plugins.requestAuth.reply);
```

Comment:
yes - I've added `isDev` to the plugin state so that it can choose the right cookie to remove. don't want to force-logout from both prod and dev if it's not necessary!